src/ConsoleGeom/ConsoleGeom/ClassEvent.cs

```diff
@@ -1,36 +1,29 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace ConsoleGeom
 {
     class ClassEvent
     {
-        public int ChangedLine()
+        public double ChangedLine()
         {
-            string a;
-            int b = 0; bool TRUE = false;
+            string parameterGet;
+            double parameterSet = 0;
+            bool checkParameter = false;
             do
             {
-                a = Console.ReadLine();
-                foreach (char c in a)
+                parameterGet = Console.ReadLine();
+                try
```

Comment:
we can not implement the program logic as try-catch blocks, please use if-else structure. Try/Catch should be used when you validate but some strange thing happens and something causes an error so you can handle it more gracefully.